lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -28,7 +28,7 @@ export class SilentIframeClient extends StandardInteractionClient {
      * @param request
      */
     async acquireToken(request: SsoSilentRequest): Promise<AuthenticationResult> {
-        this.logger.verbose("acquireTokenByIframe called");
+        this.logger.verbose("acquireTokenByIframe called", request.correlationId);
```

Comment:
Passing in the correlationId here is redundant. The logger should get reinitialized with the correlationId in the constructr